permissions/plugin-ui-samples-plugin.yml

```diff
@@ -1,10 +1,10 @@
 ---
 name: "ui-samples-plugin"
```

Comment:
I've kept this in place in case we do an empty plugin release to add a dep on design library to migrate over but not sure if needed

Reply:
Does that work if the artifact ID changes?

Reply:
would do it based on the tip of trunk before changes if we did it